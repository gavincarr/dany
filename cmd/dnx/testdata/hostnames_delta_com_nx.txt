appnet.delta.com
atl-smtp-out.delta.com
author.delta.com
del.delta.com
dev-voice.delta.com
dev.delta.com
dev.tellme.delta.com
