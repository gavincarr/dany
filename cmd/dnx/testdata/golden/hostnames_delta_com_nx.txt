appnet.delta.com
appstore-si.delta.com
appstore-uat.delta.com
appstore.delta.com
atl-smtp-out.delta.com
author.delta.com
avedge.cfitrl.delta.com
avedge.delta.com
b.e.delta.com
c.mx.e.delta.com
del.delta.com
dev-voice.delta.com
dev.delta.com
dev.tellme.delta.com
